noetic/ros-opencog/Dockerfile

```diff
@@ -113,7 +115,7 @@ WORKDIR /opencog
 # twigging this date will update the opencog sources.
 ENV LAST_SOFTWARE_UPDATE 2016-03_13
 
-# Git pull for all packages
+# Git pull for all packfages
 RUN cd /opencog/ && find . -maxdepth 1 -mindepth 1 -type d \
```

Comment:
Umm

Reply:
thx. I removed the keystroke. Regarding "mvn", I'm disabling the java tests because in general there is a bug in the test code relating to reflection access being flagged as an error in newer java versions. It will take re-writing those tests to comply with the newer java version. I re-ran all, it seems to build even better now. "link-grammar" needed flex which I hadn't noticed initially.